activesupport/lib/active_support/testing/parallelization/worker.rb

```diff
@@ -69,6 +69,9 @@ def safe_record(reporter, result)
               Minitest::UnexpectedError.new(error)
             end
             @queue.record(reporter, result)
+          rescue Interrupt => e
+            @queue.interrupt
+            raise e
```

Comment:
Calling raise inside a rescue will reraise the same error:
```suggestion
            raise
```

Reply:
You're right, thank you!